lib/perl/Genome/Model/Build/ReferenceSequence/AlignerIndex.pm

```diff
@@ -111,9 +111,11 @@ sub get_or_create {
 
     my @objects = $class->SUPER::get_or_create(@_);
 
+    my %params = @_;
```

Comment:
It's bordering on nitpicking but is this here because the above is changing `@_`?  If not how about put this line as second line of function and pass `%params` to the `get_or_create`?